service/local-dev/run_postgres.sh

```diff
@@ -10,7 +10,7 @@ start() {
     echo "starting up postgres container..."
     BASEDIR=$(dirname "$0")
     docker create --name $CONTAINER --rm -e POSTGRES_PASSWORD=password -p "$POSTGRES_PORT:5432" postgres:$POSTGRES_VERSION
-    docker cp $BASEDIR/local-postgres-init.sql $CONTAINER:/docker-entrypoint-initdb.d/docker_postgres_init.sql
+    docker cp $BASEDIR/../../scripts/postgres-init.sql $CONTAINER:/docker-entrypoint-initdb.d/docker_postgres_init.sql
```

Comment:
This file was identical to the other one, so removing the redundancy.

Reply:
They could be, but I thought a major part of having the standard scripts is that they are a facade over any repo-specific scripts/implementations. So I don't see a problem in just leaving these in their current directory.